ethereum/contracts/LightClientBridge.sol

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
can we rename this file to PolkadotRelayChainBridge to be more clear?

Reply:
Integrated latest contracts from https://github.com/Snowfork/verification-solidity/pull/12 in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/25329c8a44d706bcff17036902b25ffca55cd127 and renamed the contract in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/83e1d9bf40dc417d18a9fa7b9ee67fab01b52513